docs/architecture/seo.md

```diff
@@ -82,23 +82,21 @@ Spartacus provides a special resolver for pages that require a specific heading.
 
 However, such a title does not look good in the UI, so a different title is used for that. To support flexibility, Spartacus uses a specific `PageHeadingResolver` that can be implemented in the page resolving logic.
 
-
 ### Description Resolver
 
 Each page on the storefront can contain a `description` tag. The description tag is used in the search engine result page to improve the click-through-rate (CTR). It is not used to improve the page ranking. It is generally considered best practice to create a description tag for each page, although there are occasions when the search engine is more capable of generating the description based on the context.
 
-
 ### Image Resolver
 
-In order to share pages with social media, such as facebook, twitter, pinterest, etc. it is important to provide the right image in the metatags. The Open Graph standard from Facebook is widely adopted for this purpose. The following tag can be used to tell social media to use a specific image:
+To share pages with social media, such as Facebook, Twitter, Pinterest, and so on, it is important to provide the correct image in the meta tags. The Open Graph standard from Facebook is widely adopted for this purpose. The following tag can be used to tell social media to use a specific image:
 
 ```html
  <meta name="og:image" value="https:storefont.com/myimage">
  ```
 
-While it is possible to provide multipe images by replicating the tag with different values (i.e. for a gallery fo images), we only provide a solution for a single image. This is best practice for commerce storefronts. 
+While it is possible to provide multiple images by replicating the tag with different values (for a gallery of images, for example), Spartacus only provides a solution for a single image. This is considered best practice for commerce storefronts.
 
-The `PageImageResolver` can be used to provide a specific image for a specific page. The `ProductPageMetaResolver` has implemented the main product image url for the PDP page. Customers can implement other page providers using the `PageImageResolver` interface. 
+You can use the `PageImageResolver` to provide a specific image for a specific page. The `ProductPageMetaResolver` implements the main product image URL for the product details page. You can implement other page providers using the `PageImageResolver` interface.
```

Comment:
Similar to not using "we" in the documentation, I wonder if we should use "you"?

Reply:
What about: 
The `PageImageResolver` can be implemented to resolve a specific image for a specific page. The `ProductPageMetaResolver` demonstrates an implementation of the `PageImageResolver`, by adding the providing the main product image URL for the product details page.

Reply:
I agree, we should definitely not use "we", but the use of "you" is actively encouraged. Otherwise we fall into "passive voice", which is actively discouraged in our writers' style guide. For example, in "a sentence such as "A fixed discount is applied to the cart”, we do not know what or who applies the discount. Sentences written using passive voice are also often longer, and harder to interpret" (quote from our writers' style guide: https://help.sap.com/viewer/DRAFT/07903d1252d9446895e62b6a353c2c67/SHIP/en-US/0f4cf689f8bf414e99362c3b6348e1a8.html).

Having said that, I really like the added clarifications and details in the sentences you proposed, so if I may, I would take them and slightly modify as follows: 

"You can implement the `PageImageResolver` to resolve a specific image for a specific page. The `ProductPageMetaResolver` demonstrates an implementation of the `PageImageResolver` by providing the main product image URL for the product details page."

Reply:
There you go, I love to learn things like that (as a side project ;) ). 
All good with that change, please make the change and I'll approve accordingly.

Reply:
Nice!! 😄 
I have updated the text as discussed. Setting this conversation to resolved 🙂